ROADMAP.md

```diff
@@ -10,75 +10,87 @@ We use the [milestone](https://github.com/rook/rook/milestones) feature in Githu
-    * Disk management (adding, removing, and replacing disks) [#1435](https://github.com/rook/rook/issues/1435)
-  * File
-    * NFS Ganesha CRD [#1799](https://github.com/rook/rook/issues/1799)
-    * Dynamic Volume Provisioning for CephFS [#1125](https://github.com/rook/rook/issues/1125)
-  * Object
-    * Multi-site configuration [#1584](https://github.com/rook/rook/issues/1584)
-  * Mgr and plugins
-    * Placement group balancer support (enable the mgr module)
+  * Add support for Ceph Nautilus
+  * Beta release of CSI plug-in [#1385](https://github.com/rook/rook/issues/1385)
+    * Dynamic volume resizing [#1169](https://github.com/rook/rook/issues/1169)
+  * Automatically detect new nodes [#2208](https://github.com/rook/rook/issues/2208)
+  * Disk management (adding, removing, and replacing disks) [#1435](https://github.com/rook/rook/issues/1435)
+  * Create NFS Ganesha CRD [#1799](https://github.com/rook/rook/issues/1799)
+  * Document metadata backup and disaster recovery [#592](https://github.com/rook/rook/issues/592)
 * CockroachDB
-  * Secure deployment using certificates [#1809](https://github.com/rook/rook/issues/1809)
   * Helm chart deployment [#1810](https://github.com/rook/rook/issues/1810)
 * EdgeFS
   * Declare EdgeFS CRDs to be Beta v1 [#2506](https://github.com/rook/rook/issues/2506)
   * Automatic host validation [#2409](https://github.com/rook/rook/issues/2409)
-  * Target VDEVs
-    * Support for flexible write I/O “sync” option [#2367](https://github.com/rook/rook/issues/2367)
+  * Allow for more performance vs. availability tradeoff configuration [#2367](https://github.com/rook/rook/issues/2367)
   * Object
     * OpenStack/SWIFT CRD [#2509](https://github.com/rook/rook/issues/2509)
     * Support for S3 bucket as DNS subdomain [#2510](https://github.com/rook/rook/issues/2510)
-  * Block (iSCSI)
-    * Support Block CSI [#2507](https://github.com/rook/rook/issues/2507)
-  * Mgr
-    * Support for Prometheus Dashboard and REST APIs [#2401](https://github.com/rook/rook/issues/2401)
-    * Support for Management GUI with automated CRD wizards [#2508](https://github.com/rook/rook/issues/2508)
+  * Support Block (iSCSI) CSI [#2507](https://github.com/rook/rook/issues/2507)
+  * Support for Prometheus Dashboard and REST APIs [#2401](https://github.com/rook/rook/issues/2401)
+  * Support for Management GUI with automated CRD wizards [#2508](https://github.com/rook/rook/issues/2508)
+  * Failure domains and zoning provisioning support [#2513](https://github.com/rook/rook/issues/2513)
+  * Multi-Namespace clusters support [#2878](https://github.com/rook/rook/issues/2878)
+  * Embedded mode support [#2810](https://github.com/rook/rook/issues/2810)
 * Minio
   * Helm chart deployment [#1814](https://github.com/rook/rook/issues/1814)
+
+
+## Initial planning for Rook 1.1
+
+* Allow Ceph disk selection by full path [#2845](https://github.com/rook/rook/pull/2845)
```

Comment:
Given #2845 is about ready to merge, i'd expect it in 1.0

Reply:
I'm still hesitant about adding #2845 to the Roadmap for v1.0 even though it looks like it might merge soon. IMO, we are past the point where we should have been feature locked for v1.0, and we should be now focusing on bug fixes and documentation. I see the possibility that #2845 makes it into v1.0, but I don't think it's wise to add it to the plan so late; I wouldn't hold up the v1.0 release to get it in.